plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -7891,6 +7891,13 @@ public void testOptimize()
         assertUpdate(optimizeEnabledSession, "ALTER TABLE " + tableName + " EXECUTE optimize(file_size_threshold => '10B')");
         assertThat(getTableFiles(tableName)).hasSameElementsAs(compactedFiles);
 
+        // optimize with delimited procedure name
```

Comment:
This is just a smoke test coverage. I put the together because it's not a detailed test for ALTER TABLE EXECUTE itself. (such test doesn't exist, and I don't want to add it)